src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/basic/DogFacts.java

```diff
@@ -0,0 +1,76 @@
+package com.programmerdan.minecraft.simpleadminhacks.hacks.basic;
+
+import com.programmerdan.minecraft.simpleadminhacks.SimpleAdminHacks;
+import com.programmerdan.minecraft.simpleadminhacks.framework.BasicHack;
+import com.programmerdan.minecraft.simpleadminhacks.framework.BasicHackConfig;
+import com.programmerdan.minecraft.simpleadminhacks.framework.autoload.AutoLoad;
+import java.util.List;
+import java.util.UUID;
+import net.md_5.bungee.api.ChatColor;
+import net.md_5.bungee.api.chat.TextComponent;
+import org.bukkit.Bukkit;
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import vg.civcraft.mc.civmodcore.playersettings.PlayerSettingAPI;
+import vg.civcraft.mc.civmodcore.playersettings.gui.MenuSection;
+import vg.civcraft.mc.civmodcore.playersettings.impl.BooleanSetting;
+import vg.civcraft.mc.civmodcore.util.ConfigParsing;
+
+public class DogFacts extends BasicHack {
+
+	@AutoLoad
+	private List<String> announcements;
+	@AutoLoad
+	private String intervalTime;
+	private long interval;
+	private BooleanSetting disableAnnouncements;
+	private int counter = 0;
+	private int tickOffset;
+
+	public DogFacts(SimpleAdminHacks plugin, BasicHackConfig config) {
+		super(plugin, config);
+		this.interval = ConfigParsing.parseTime(intervalTime);
```

Comment:
This can not work, the member variables can not be set already before the constructor call (which creates the object)